### PR TITLE
macOS Catalina [bug] #16

### PR DESCRIPTION
**Pull request by JoNilsson**
Okay, think this was an easy fix, though deff want to look it over because i am a total n00b to Rust.  
_This is the first Rust repo i've contributed to._ 
tested locally, seems to be working great again and that is great news for me.  Was bummed to not have my fav pass-manager after updating to Catalina.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,11 +1,16 @@
 [package]
 name = "passbase"
-version = "0.1.0"
-authors = ["dev@ojford.com"]
+version = "0.1.1" # patch; macOS Catalina filesystem fixes
+authors = ["Oliver Ford <dev@ojford.com>", "Johanness Nilsson <jnilsson@teknomadix.com>"]
+description = "Passbase is a free password manager built on Keybase. Generate long, random, secure passwords and store encrypted copies of them on your Keybase File System."
+documentation = "https://github.com/OJFord/passbase/blob/master/README.md"
+homepage = ".https://github.com/OJFord/passbase"
```
Comment (JoNilsson) [inline Cargo.toml]: small `.` typo here

**Comment by OJFord** [comment]
Hi @JoNilsson, I'm really sorry it took me so long to get to this.

I haven't accepted this PR mainly because I wanted to keep support for older versions of macOS, as well as other OSs, but there are some other improvements/tidy-ups in here that I'd be glad to review if you wanted to rebase and provide separately.

The title support should now be present in master (8bb8ef525cc229d08a94696295800ae5466176c2), and I'll remind myself how I released it previously (it's been so long!) now.

Many thanks!

**Comment by JoNilsson** [comment]
> I wanted to keep support for older versions of macOS
Ah,  yeah, that's an obvious need that I had overlooked.  Very good point.  I'll remember to keep that in mind in the future.  
Been using #17 PR since it was put in.  
I'm going to switch back to the main repo now and test,  on Catalina public beta.
Since I put in the PR, the old KBFS has been archived and Keybase has pushed out a helper that installs `Mac-FUSE`,   but it should have no effect on how `passbase` interacts with Catalina.

**Comment by OJFord** [comment]
> Since I put in the PR, the old KBFS has been archived and Keybase has pushed out a helper that installs Mac-FUSE, but it should have no effect on how passbase interacts with Catalina.

Good point - I'd forgotten about that. You have to accept permissions for it (because it's closed-source; I did it a couple of weeks ago when I upgraded) but other than that, that's what's running on mine now so it should be okay 🤞.

**Pull request closed without merging**